inputfiles/overridingTypes.jsonc

```diff
@@ -2479,6 +2479,13 @@
                             }
                         }
                     }
+                },
+                "iterator": {
+                    "type": {
+                        "0": {
+                            "overrideType": "R"
```

Comment:
This seems redundant here.

Reply:
Fair enough. I'll need to remember to add it back once browsers *actually* add support for this async iterator, and it actually ends up in the generated types. 😛

Reply:
Hmmmmm. That said, this is the only needed manual data and we know it will eventually come, so maybe we can keep this 👀